mysql-test/suite/galera/r/galera_bf_abort.result

```diff
@@ -4,7 +4,7 @@ START TRANSACTION;
 INSERT INTO t1 VALUES (1);
 INSERT INTO t1 VALUES (1);
 INSERT INTO t1 VALUES (2);
-ERROR 40001: Deadlock found when trying to get lock; try restarting transaction
+ERROR 40001: WSREP cancelled the transaction
```

Comment:
From what I can gather, this is possible in some places (may be able to detect BFA or LCF).